<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="pad">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>216</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QLabel" name="border">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>216</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
    background-color: rgb(71, 71, 71);
    border: 1px solid black
}QComboBox QAbstractItemView {
    border: 1px solid black;
    background: rgb(71, 71, 71);
    selection-background-color: gray;
}QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 19px;
    border-right-width: 1px;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;}</string>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="settings_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="export_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="import_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reload_btn">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="debug_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug Mode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="refresh_rate">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>100</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
  </widget>
  <widget class="QLabel" name="refresh_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="ui_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>UI Skin</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ui_skin">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>120</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="tutorial_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Link OCR Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="speed">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Average speed:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
